### PR TITLE
Dec heartbeat

### PR DESCRIPTION
**Pull request by jendefig**
Main image is still to be adjusted.  Don't want 2 blue backgrounds next to each other on the main page.

- [x]  There are no grammatical errors, jargon, or obvious stylistical issues.
- [x]  The flow and messaging corresponds to what has been defined in the Phase 3.
- [x]  Have images been run through tiny.png?
- [x]  Are all lines 80 characters or less?
- [x]  Are Oxford commas observed?
- [x]  Content Creator has done complete review for typos/language after taking break from content
- [x]  links checked in piece
- [x]  Have reviewed what the piece looks like on your own [machine using yarn develop](https://dvc.org/doc/user-guide/contributing/docs#development-environment)

### CODE REVIEW

**Comment by jendefig** [comment]
@MKhalusova This is ready for review!

File: content/blog/2021-12-15-december-21-heartbeat.md [hunk 1]
```diff
@@ -0,0 +1,269 @@
...
+## DVC + CML + RASA = ❤️
+
+[**Matthew Upson**](https://twitter.com/m_a_upson), Founder at
+[MantisNLP,](https://mantisnlp.com/) an AI consultancy focused on NLP, along
+with his team, put out the
+[first blog post](https://medium.com/mantisnlp/mlops-for-conversational-ai-with-rasa-dvc-and-cml-part-i-beec756e8e7f)
+in a series showing how to use DVC and CML along with Rasa in developing
+conversational AI. This post sets the scene for the following more detailed
+parts, but lays out DVC's use for generating the DAG as well as logging metrics
+and CML to do the testing. We're looking forward to the next installments!
```
Comment (MKhalusova) [inline content/blog/2021-12-15-december-21-heartbeat.md:54]: "and using CML" maybe?

File: content/blog/2021-12-15-december-21-heartbeat.md [hunk 2]
```diff
@@ -0,0 +1,269 @@
...
+develop a pipeline
+([Source link](https://mattsegal.dev/devops-academic-research.html))_
+
+## Are you confused by how many MLOps tools there are?
+
+Well
+[Thoughtworks](https://www.thoughtworks.com/?utm_source=google-search&utm_medium=paid-media&utm_campaign=always-on-brand_2021-11&utm_term=thoughtworks&utm_content=RSAad1&gclid=Cj0KCQiA2NaNBhDvARIsAEw55hg2li5srltu8ppVsxLzcnv-pYWRmvnCk_jmljiC2ocyM4tc7EUEt9gaAoVWEALw_wcB)
+included DVC in its recent
+[Thoughtworks Guide to MLOps Platforms](https://www.thoughtworks.com/what-we-do/data-and-ai/cd4ml/guide-to-evaluating-mlops-platforms).
+While being included is great, things move so fast that they seemed to miss our
```
Comment (MKhalusova) [inline content/blog/2021-12-15-december-21-heartbeat.md:104]: Personally, I would say "they seem to have missed". Leaving it up to you though.

**Review by MKhalusova — APPROVED** [review]
Great job on the Heartbeat, Jeny! I've left a couple of minor comments for your consideration. They are not blockers though.

**Review by shcheklein — APPROVED** [review]
Looks great to me! 

One thing to try is wrapping images

``

`:wrap-left`

I see that we don't use this feature, was wondering if the team is familiar with it. It can make content a bit more diverse.

Also, "Frame 49" - is it expected on the top blog image, or an error of exporting from some visual tool? :) just to be sure

**Comment by jendefig** [comment]
> Looks great to me!
> 
> One thing to try is wrapping images
> 
> ``
> 
> `:wrap-left`
> 
> I see that we don't use this feature, was wondering if the team is familiar with it. It can make content a bit more diverse.
> 
> Also, "Frame 49" - is it expected on the top blog image, or an error of exporting from some visual tool? :) just to be sure

On the wrap left, just not familiar will give a try.  
On the Frame 49 - this blog image is a placeholder - just a screen shot.  I'm having Roman change the background so we don't have back to back blue backgrounds in the blog.

File: content/blog/2021-12-15-december-21-heartbeat.md (hunk 2 above)
Comment (jendefig) [reply content/blog/2021-12-15-december-21-heartbeat.md:104]: Yep.  That's better.  Thanks!

**Comment by jendefig** [comment]
@shcheklein This is ready to go. I added one left wrap.  Went over with @julieg18.  She helped me with some issues with that and yarn develop.   Will definitely use the wrap when it works!  Also corrected @MKhalusova's edits and tinified the correct blog image which is now in place. 👍

**Comment by shcheklein** [comment]
@jendefig merged! good stuff 🎉

**Pull request merged**